tests/account_tests.rs

```diff
@@ -1,18 +1,424 @@
 use binance::api::*;
 use binance::config::*;
 use binance::account::*;
+use binance::model::*;
 
 #[cfg(test)]
 mod tests {
     use super::*;
     use mockito::{mock, Matcher};
+    use float_cmp::*;
 
     #[test]
-    fn test_order_market_buy_using_quote_quantity() {
-        let mock_exchange_info = mock("POST", "/api/v3/order")
+    fn get_account() {
+
+        let mock_get_account = mock("GET", "/api/v3/account")
+            .with_header("content-type", "application/json;charset=UTF-8")
+            .match_query(Matcher::Regex("recvWindow=1234&timestamp=\\d+&signature=.*".into()))
+            .with_body_from_file("tests/mocks/account/get_account.json")
+            .create();
+
+        let config = Config::default()
+            .set_rest_api_endpoint(mockito::server_url())
+            .set_recv_window(1234);
+        let account: Account = Binance::new_with_config(None, None, &config);
+        let _ = env_logger::try_init();
+        let account = account.get_account().unwrap();
+
+        mock_get_account.assert();
+
+        assert!(approx_eq!(f32, account.maker_commission, 15.0, ulps = 2));
+        assert!(approx_eq!(f32, account.taker_commission, 15.0, ulps = 2));
+        assert!(approx_eq!(f32, account.buyer_commission, 0.0, ulps = 2));
+        assert!(approx_eq!(f32, account.seller_commission, 0.0, ulps = 2));
+        assert_eq!(account.can_trade, true);
+        assert_eq!(account.can_withdraw, true);
+        assert_eq!(account.can_deposit, true);
+
+        assert!(!account.balances.is_empty());
+
+        let first_balance = &account.balances[0];
+        assert_eq!(first_balance.asset, "BTC");
+        assert_eq!(first_balance.free, "4723846.89208129");
+        assert_eq!(first_balance.locked, "0.00000000");
+
+        let second_balance = &account.balances[1];
+        assert_eq!(second_balance.asset, "LTC");
+        assert_eq!(second_balance.free, "4763368.68006011");
+        assert_eq!(second_balance.locked, "0.00000000");
+
+    }
+
+    #[test]
+    fn get_balance() {
+
+        let mock_get_account = mock("GET", "/api/v3/account")
+            .with_header("content-type", "application/json;charset=UTF-8")
+            .match_query(Matcher::Regex("recvWindow=1234&timestamp=\\d+&signature=.*".into()))
+            .with_body_from_file("tests/mocks/account/get_account.json")
+            .create();
+
+        let config = Config::default()
+            .set_rest_api_endpoint(mockito::server_url())
+            .set_recv_window(1234);

```

Comment:
I put this `recv_window=1234` in my first test case, maybe it's not needed in all of them.

Reply:
I think all signed methods require it, no?